spec/20-http_header_format.md

```diff
@@ -277,13 +277,13 @@ The key is an identifier that describes the vendor.
 
 ``` abnf
 key = lcalpha 0*255( lcalpha / DIGIT / "_" / "-"/ "*" / "/" )
-key = lcalpha 0*240( lcalpha / DIGIT / "_" / "-"/ "*" / "/" ) "@" lcalpha 0*13( lcalpha / DIGIT / "_" / "-"/ "*" / "/" )
+key = ( lcalpha / DIGIT ) 0*240( lcalpha / DIGIT / "_" / "-"/ "*" / "/" ) "@" lcalpha 0*13( lcalpha / DIGIT / "_" / "-"/ "*" / "/" )
 lcalpha    = %x61-7A ; a-z
 ```
 
-**Note**: Identifiers MUST begin with a lowercase letter, and can only contain lowercase letters (`a`-`z`), digits (`0`-`9`), underscores (`_`), dashes (`-`), asterisks (`*`), and forward slashes (`/`).
+**Note**: Identifiers MUST begin with a lowercase letter or a digit, and can only contain lowercase letters (`a`-`z`), digits (`0`-`9`), underscores (`_`), dashes (`-`), asterisks (`*`), and forward slashes (`/`).
```

Comment:
I was trying to wrap my head around this and think there's a doc glitch here.

this phrase 'or a digit' here should apply both to simple and multi-tenant encodings. However, line 279 wasn't updated so basically the text doesn't match the abnf.

I think either the relaxing should apply in both abnf (ex also 279) or this special case about "or a digit" should move to the next paragraph which talks about the second encoding.

Reply:
https://github.com/w3c/trace-context/pull/386